src/com/facebook/buck/python/CxxPythonExtensionDescription.java

```diff
@@ -198,10 +200,16 @@ protected Path getExtensionPath(BuildTarget target, Flavor pythonPlatform, Flavo
                 args.platformLinkerFlags,
                 cxxPlatform)));
 
-    // Embed a origin-relative library path into the binary so it can find the shared libraries.
+    // Embed a extension-relative library path into the .so so it can find the shared libraries that
+    // it depends on.
+    String rootPath = MorePaths.relativize(
+        Paths.get(""), Paths.get(args.baseModule.or(""))).toString();
```

Comment:
These aren't real paths - this attempts to get enough `../` so that a module placed in a directory specified in `args.baseModule` would find the "root".